stabilize.md

```diff
@@ -0,0 +1,180 @@
+
+    syntax FlapStep ::= FlapAuthStep
+ // --------------------------------
+
+    syntax FlapAuthStep ::= AuthStep
+ // --------------------------------
+
+    syntax FlapAuthStep ::= WardStep
+ // --------------------------------
+
+    syntax FlapAuthStep ::= "init" Address Address
+ // ----------------------------------------------
+
+    syntax FlapStep ::= StashStep
+ // -----------------------------
+
+    syntax FlapStep ::= ExceptionStep
+ // ---------------------------------
+
+    syntax FlapStep ::= "kick" Int Int
+ // ----------------------------------
+
+    syntax FlapStep ::= "tend" Int Int Int
+ // --------------------------------------
+
+    syntax FlapStep ::= "deal" Int
+ // ------------------------------
+
+    syntax FlapStep ::= "cage" Int
+ // ------------------------------
+
+    syntax FlapStep ::= "yank" Int
+ // ------------------------------
+```
+
+Flop Semantics
+--------------
+
+```k
+    syntax MCDStep ::= "Flop" "." FlopStep
+ // --------------------------------------
+
+    syntax FlopStep ::= FlopAuthStep
+ // --------------------------------
+
+    syntax FlopAuthStep ::= AuthStep
+ // --------------------------------
+
+    syntax FlopAuthStep ::= WardStep
+ // --------------------------------
+
+    syntax FlopAuthStep ::= "init" Address Address
+ // ----------------------------------------------
+
+    syntax FlopStep ::= StashStep
+ // -----------------------------
+
+    syntax FlopStep ::= ExceptionStep
+ // ---------------------------------
+
+    syntax FlopStep ::= "kick" Int Int Int
+ // --------------------------------------
+
+    syntax FlopStep ::= "tick" Int
+ // ------------------------------
+
+    syntax FlopStep ::= "dent" Int Int Int
+ // --------------------------------------
+
+    syntax FlopStep ::= "deal" Int
+ // ------------------------------
+
+    syntax FlopStep ::= "cage"
+ // --------------------------
+
+    syntax FlopStep ::= "yank" Int
+ // ------------------------------
+```
+
+Vow Semantics
+-------------
+
+```k
+    syntax MCDStep ::= "Vow" "." VowStep
+ // ------------------------------------
+
+    syntax VowStep ::= VowAuthStep
+ // ------------------------------
+
+    syntax VowAuthStep ::= AuthStep
+ // -------------------------------
+
+    syntax VowAuthStep ::= WardStep
+ // -------------------------------
+
+    syntax VowAuthStep ::= "init" Address Address Address
+ // -----------------------------------------------------
+
+    syntax VowStep ::= StashStep
+ // ----------------------------
+
+    syntax VowStep ::= ExceptionStep
+ // --------------------------------
+
+    syntax VowStep ::= "fess" Int
+ // -----------------------------
+
+    syntax VowStep ::= "flog" Int
+ // -----------------------------
+
+    syntax VowStep ::= "heal" Rad
+ // -----------------------------
+
+    syntax VowStep ::= "kiss" Rad
+ // -----------------------------
+
+ // TODO: "flop" and "flap" seem to conflict with the config cell names
```

Comment:
There's either a way around this in K or we need to rename something. I say renaming the configuration cells is preferred since then the user can use Vow.flop/Vow.flap as expected.